apps/ios/SimpleXChat/ChatTypes.swift

```diff
@@ -2245,7 +2250,16 @@ public struct ChatItem: Identifiable, Decodable {
     public var memberDisplayName: String? {
         get {
             if case let .groupRcv(groupMember) = chatDir {
-                return groupMember.chatViewName
+                switch content {
+                case let .rcvGroupEvent(rcvGroupEvent: .memberProfileUpdated(fromProfile, toProfile)):
+                    if toProfile.displayName != fromProfile.displayName || toProfile.fullName != fromProfile.fullName {
+                        return nil
```

Comment:
remove `returns` (and `get`, as it's rewritten anyway)

Reply:
the first if can be ternary

Reply:
again ternary would make everything a single unindented block harder to understand what's a condition and what's a value